ios/Video/Features/RCTVideoUtils.swift

```diff
@@ -9,7 +9,16 @@ enum RCTVideoAssetsUtils {
         for mediaCharacteristic: AVMediaCharacteristic
     ) async -> AVMediaSelectionGroup? {
         if #available(iOS 15, tvOS 15, visionOS 1.0, *) {
-            return try? await asset?.loadMediaSelectionGroup(for: mediaCharacteristic)
+            // swiftlint:disable shorthand_optional_binding
+            guard let asset = asset else {
+                return nil
+            }
+            if let mediaSelectionGroup = try? await asset.loadMediaSelectionGroup(for: mediaCharacteristic) {
+                return mediaSelectionGroup
+            } else {
+                return nil
+            }
+            // swiftlint:enable shorthand_optional_binding
```

Comment:
@Fausto95 How about this ?
```suggestion
            do {
                guard let asset else {
                    return nil
                }

                return try await asset.loadMediaSelectionGroup(for: mediaCharacteristic)
            } catch {
                return nil
            }
```

Reply:
Yeah, should work too..
I'll apply the changes and see if the linter won't error.